### PR TITLE
Add support to annotate AlertRules

### PR DESCRIPTION
**Pull request by mhutter**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Update the ./CHANGELOG.md.
- [x] Link this PR to related issues.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,16 +10,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
 - Upgrade kube-prometheus ([#6])
 
 ### Fixed
 
 - Update kube-prometheus version locks to include recent dependency fixes ([#4]).
 
-[Unreleased]: https://github.com/projectsyn/component-rancher-monitoring/compare/084a263baf909b627d2861790806ac8f7de3f580...HEAD
-
+[unreleased]: https://github.com/projectsyn/component-rancher-monitoring/compare/084a263baf909b627d2861790806ac8f7de3f580...HEAD
```
Comment (srueg) [inline CHANGELOG.md:24]: Not sure if this is case sensitive or not...

File: component/rules.jsonnet [hunk 2]
```diff
@@ -2,6 +2,10 @@ local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
 local params = inv.parameters.rancher_monitoring;
 local ignoreNames = params.alerts.ignoreNames;
+local customAnnotations = params.alerts.customAnnotations;
+local defaultAnnotations = {
+  syn_component: 'rancher-monitoring',
```
Comment (srueg) [inline component/rules.jsonnet:7]:
```suggestion
  syn_component: inv.parameters._instance,
```

the `_instance` parameter is set to the component name (if not multi instantiated) see https://github.com/projectsyn/documentation/blob/master/docs/modules/SDDs/pages/0025-commodore-component-instantiation.adoc for further details

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,16 +10,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Initial open-source implementation ([#1])
+- Annotate all alerts with the Syn component name
```
Comment (srueg) [inline CHANGELOG.md:13]:
```suggestion
- Annotate all alerts with the Syn component name ([#8])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,16 +10,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Allow configuring custom annotations via inventory
```
Comment (srueg) [inline CHANGELOG.md:14]:
```suggestion
- Allow configuring custom annotations via inventory ([#8])
```

**Review by srueg — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mhutter) [reply CHANGELOG.md:24]: Unrelated change by my autoformatter 🙈  will revert

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,16 +10,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ### Fixed
 
 - Update kube-prometheus version locks to include recent dependency fixes ([#4]).
 
 [Unreleased]: https://github.com/projectsyn/component-rancher-monitoring/compare/084a263baf909b627d2861790806ac8f7de3f580...HEAD
-
 [#1]: https://github.com/projectsyn/component-rancher-monitoring/pull/1
 [#4]: https://github.com/projectsyn/component-rancher-monitoring/pull/4
 [#6]: https://github.com/projectsyn/component-rancher-monitoring/pull/6
```
Comment (srueg) [inline CHANGELOG.md:27]:
```suggestion
[#6]: https://github.com/projectsyn/component-rancher-monitoring/pull/6
[#8]: https://github.com/projectsyn/component-rancher-monitoring/pull/8
```

**Review by srueg — APPROVED** [review]

**Pull request merged**